frontend/src/store/shoot/helper.js

```diff
@@ -486,6 +488,9 @@ export function getSortVal (context, item, sortBy) {
         name: metadata.name,
       })
     }
+    case 'workers': {
+      return value === 'workerless' ? 0 : value
```

Comment:
? see above

Reply:
in case of sorting we want to have zero to make the sorting work. `workerless`value is used for searching, see comment above.
If we want to have this cleaner, we could introduce a "getSearchVal" function, which like the `getSortVal` function enriches the value in case of `workerless`